app/models/ability.rb

```diff
@@ -34,8 +34,8 @@ def not_signed_in
       conference.splashpage && conference.splashpage.public == true
     end
     # Can view the schedule
-    can [:schedule, :events], Conference do |conference|
-      conference.program.cfp && conference.program.schedule_public
+    can [:show, :schedule, :events], Conference do |conference|
```

Comment:
`schedules_controller ` load_authorize conference object for use in schedules show action. To make schedule show action accessible to user who is not logged in, conference show action is added to can in ability.rb